code/crates/core-consensus/src/effect.rs

```diff
@@ -84,6 +84,11 @@ where
     /// Resume with: [`resume::Continue`]
     Publish(SignedConsensusMsg<Ctx>, resume::Continue),
 
+    /// Publish a polka certificate to peers
+    ///
+    /// Resume with: [`resume::Continue`]
+    PublishGossipMessage(GossipMsg<Ctx>, resume::Continue),
```

Comment:
```suggestion
    PublishGossipMsg(GossipMsg<Ctx>, resume::Continue),
```

Since we use _msg_ instead of _message_ all over the codebase already

Reply:
2526b2c017db83ed680f800c59a6efea783246d1